apps/full-site-editing/full-site-editing-plugin/donations/blocks/donation/edit.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __experimentalBlock as Block, InnerBlocks } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import intervalClassNames from './interval-class-names';
+
+const Edit = ( { attributes, context } ) => {
+	const { interval } = attributes;
+	const currency = context[ 'donations/currency' ];
+	const showCustom = context[ 'donations/showCustom' ];
+
+	const headers = {
+		'one-time': __( 'Make a one-time donation', 'full-site-editing' ),
+		'1 month': __( 'Make a monthly donation', 'full-site-editing' ),
+		'1 year': __( 'Make a yearly donation', 'full-site-editing' ),
+	};
+
+	const buttons = {
+		'one-time': __( 'Donate', 'full-site-editing' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **79** times:
`translate( 'Donate' )` *ES Score: 13*